### ISSUE TITLE
After loading the save file, the box does not show a blue circle

### ISSUE THREAD

**Issue by hjyoon** [opennox/opennox#564]
- new game
![image](https://user-images.githubusercontent.com/16138767/203822101-bc811bdd-831c-4886-b7b7-fd789dd91c42.png)

- after loading save file
![image](https://user-images.githubusercontent.com/16138767/203821731-a35fbcc0-2531-4a90-ab53-24001b40948b.png)

**Comment by Evengard** [comment]
I think we had an issue for this sort of problems... Let me check.

**Comment by Evengard** [comment]
Yep, it's a duplicate of https://github.com/noxworld-dev/opennox/issues/535

**Comment by dennwc** [comment]
This may be something different. In #535 it happens randomly, while here it can be reproduced by saving and then loading. Need to check it just in case.

**Comment by Evengard** [comment]
Nah, I think it's just a stable repro for the same problem. Kinda worth using that info, true, but I think the problem is the same.

**Comment by angrykirc** [comment]
This problem is different. #535 happens in vanilla Nox, and it just messes up red/blue colors of the circle, whereas this issue is about the circle vanishing completely. 
Though I think someone should check the effect not only on chests, but also on mobs to clarify

**Comment by KILAH4716** [comment]
How many times did you reload the save to get this bug? Just once?

**Comment by KILAH4716** [comment]
So I did some experimenting (and this occurs in Vanilla as well), and I've made a conclusion; I believe this mechanic is entirely intentional, and I don't think it's an issue.

I played through the Gauntlet, and opened a few chests in Dun Mir prior, and went back to every single one of those chests, and they all functioned the exact same way: their circle was removed. I believe that the circle is there exclusively so you can click the chest to interact with it and open it, but once the chest is opened, the chest is no longer important. So when you reload a game from a save or by loading and reloading a map without saving, the game does not register an opened chest as an interactable object.

This is separate from the IFF issue I found. I have not once found an issue with entities missing their targeting circle. Their circle might be the wrong color, but it's always there, and the problem occurs after several loads, not just one.

**Comment by dennwc** [comment]
So if I understood correctly, this is an issue present in vanilla. The only thing that vanilla does wrong is that it doesn't remove the interaction circle immediately and only reflects that change after map reload (either by zone transition or save/load).
